manifests/init.pp

```diff
@@ -224,9 +224,12 @@
   }
 
 
-  # Configure default gateway (On RedHat). Also hostname is set.
+  # Configure sysconfig/network (On RedHat).
   if $::osfamily == 'RedHat'
-  and $network::gateway {
+    and ($network::gateway
+    or $network::hostname
+    or $network::ipv6enable
+    or $network::nozeroconf) {
```

Comment:
The reason is that if we don't set any of those params we don't want the module to manage ```/etc/sysconfig/network``` at all.